realm/src/main/java/io/realm/Realm.java

```diff
@@ -953,7 +952,8 @@ boolean contains(Class<?> clazz) {
     }
 
     /**
-     * Get all objects of a specific Class
+     * Get all objects of a specific Class. If no objects exist, the returned RealmResults will not
+     * be null. The RealmResults.size() to check the number of objects instead.
      *
      * @param clazz the Class to get objects of
      * @return A RealmResult list containing the objects
```

Comment:
Usually you would provide such information in the @return description, something along the lines of "@return A RealmResult list containing all objects. If no objects exist, an empty list is returned".

Saying that .size() must be used is not needed as that is the standard method for lists.